### DISCUSSION TITLE
How is the latest adetailer working now if the accordeon ui is not being loaded?

### DISCUSSION THREAD

**Discussion by Tillerz** [Bing-su/adetailer#642] [Q&A]
There are UIs out there where you can just save all configs and they are reloaded on restart. Now with adetailer requiring the UI to be loaded, that is not working anymore, you need to go in there and open it. Why? That is a not well thought through implementation.

And even before the "accordeon ui needs to be active and open" change, adetailer didn't load the last configured pt files, it just defaulted back to face and stuff, despite showing a different one when you opened the ui. It showed the "fashion" pt, for example, but still fixed "faces". Only selecting a different pt and then going back to "fashion" fixed that.

There's some serious loading/activation issues going on here.

**Comment by brknsoul** [comment]
For A1111/Forge: Enable (open) adetailer, Settings, Defaults (near the bottom), View Changes, ensure that "customscript/!adetailer.py/txt2img/ADetailer/value" is listed, and Save Changes.

This will ensure adetailer is enabled on start.

**Reply by Tillerz** [reply]
I am in control tab, and this doesn't seem to work. I can sewe control/ADetailer detector/value and control/ADetailer detector 2nd/value. But when generating an image, adetailer does do nothing.
